packages/appstore/src/components/demo-reset-balance/demo-reset-balance.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Icon, Text, Button, Div100vhContainer } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { useRequest } from '@deriv/api';
+import { observer, useStore } from '@deriv/stores';
+import './demo-reset-balance.scss';
+
+type TDemoResetBalanceProps = {
+    setActiveTabIndex?: (index: number) => void;
+};
+
+const DemoResetBalance = observer(({ setActiveTabIndex }: TDemoResetBalanceProps) => {
+    const { mutate, isSuccess: isResetBalanceSuccess } = useRequest('topup_virtual');
+    const { client, ui } = useStore();
+    const { accounts, loginid } = client;
+    const { is_mobile } = ui;
+
+    const can_reset_balance = loginid && (accounts[loginid]?.balance || 0) !== 10000;
```

Comment:
shouldn't we check if the account is demo?

Reply:
@amina-deriv
we check it in `packages/appstore/src/components/modals/wallet-modal/wallet-modal-body.tsx` component on line 76
only if there is a `demo` account we will show `DemoResetBalance` component